src/protocols/rdp/guac_rdpdr/rdpdr_fs_service.c

```diff
@@ -153,5 +153,42 @@ void guac_rdpdr_register_fs(guac_rdpdrPlugin* rdpdr, char* drive_name) {
     /* Init data */
     device->data = rdp_client->filesystem;
 
+	/* Init directory for file transfer */
```

Comment:
Just off the bat, please follow the style used throughout the rest of the code - each tab should be 4 spaces.

See: http://guacamole.apache.org/guac-style/